recipes/plasmapy/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set version = "0.1.2" %}
+
+package:
+  name: plasmapy
+  version: {{ version }}
+
+source:
+  fn: plasmapy-{{version}}.tar.gz
+  url: https://pypi.io/packages/source/p/plasmapy/plasmapy-{{ version }}.tar.gz
+  sha256: 2a2bbc064a8b5726bc32447973babba9355e5c7d9483b1eebc82c6e83a9a173e
+
+build:
+  number: 0
+  script: python setup.py install --offline --single-version-externally-managed --record=record.txt
```

Comment:
Please, change that to:
```
script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```

Reply:
The original PR (#4793) did it this way (see https://github.com/conda-forge/sunpy-feedstock/blob/master/recipe/meta.yaml#L14 ), and @SolarDrew claimed needing to do it this way in the previous PR, seemingly following SunPy's example. This seemed to make sense to me, as both are taking advantage of Astropy's package ecosystem. It seems likely that this change caused the most recent build failure.

Reply:
Confirmed by a6942ed

Reply:
This is the new pattern in ``conda-forge``, that's why I requested that